src/exception/call_stack/stackwalk.cr

```diff
@@ -97,7 +96,8 @@ struct Exception::CallStack
 
       if LibC.SymGetModuleInfoW64(LibC.GetCurrentProcess, pc, module_info) != 0
         mod_displacement = pc - LibC.SymGetModuleBase64(LibC.GetCurrentProcess, pc)
-        file_name = "#{String.from_utf16(module_info.value.loadedImageName.to_unsafe)[0]}+0x#{mod_displacement.to_s(16)}"
+        image_name = String.from_utf16(module_info.value.loadedImageName.to_unsafe)[0]
+        file_name = "#{image_name} +#{mod_displacement}"
```

Comment:
IIRC the format you've used previously is common in windows C++ stacktraces, so I'd stick to it for consistency.

```suggestion
        file_name = "#{image_name}+#{mod_displacement.to_s(16)}"
```

Reply:
Windows stack traces are not an exchange format. We don't have to be consistent with it, but we can be consistent with `Exception::CallStack.print_frame`. Also ASP.NET omits `0x` too, so it is not truly a Windows thing.